testSrc/unit/io/flutter/run/daemon/DaemonEventTest.java

```diff
@@ -76,7 +76,12 @@ public void onAppProgressFinished(DaemonEvent.AppProgress event) {
 
       @Override
       public void onAppStopped(DaemonEvent.AppStopped event) {
-        logEvent(event, event.appId);
+        if (event.error != null) {
+          logEvent(event, event.appId, event.error);
+        }
+        else {
```

Comment:
yeah, this is the jetbrains style. Using it lets us have the same style as the Dart plugin code